community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/BaseStoreMigrationParticipant.java

```diff
@@ -33,28 +33,37 @@
  */
 public class BaseStoreMigrationParticipant implements StoreMigrationParticipant
 {
+    protected final String name;
+
+    public BaseStoreMigrationParticipant( String name )
```

Comment:
Or AbstractionStoreMigrationParticipant... goes in line with most other such base classes.